.github/BOTMETA.yml

```diff
@@ -1200,7 +1246,8 @@ files:
   $module_utils:
     support: community
   $module_utils/acme.py: *crypto
-  $module_utils/ansible_tower.py: *tower
```

Comment:
This dropped the `*tower`, because I was unsure if syntax would keep working if I kept it, and the other entries did not have it. I also don't want to declare things like maintainers, because we are not maintaining it.